go.mod

```diff
@@ -11,12 +11,13 @@ require (
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/go-version v1.3.0
 	github.com/hashicorp/hcl/v2 v2.8.2 // indirect
-	github.com/hashicorp/hcp-sdk-go v0.10.0
+	github.com/hashicorp/hcp-sdk-go v0.11.0
```

Comment:
Heads up, this just got bumped to v0.12.0 in https://github.com/hashicorp/terraform-provider-hcp/pull/178

Reply:
Updating, thanks !